addon/components/ember-chart.js

```diff
@@ -7,6 +7,8 @@ import Ember from 'ember';
 import layout from '../templates/components/ember-chart';
 import ChartObject from '../utils/chart-object';
 
+const { isNone, isEmpty, computed } = Ember;
```

Comment:
Would it make sense to import `get` here as well since you're doing it for the other methods?